portal/src/texts/getstarted/CodeGuide.mdx

```diff
@@ -43,3 +43,11 @@ Hvis du vil legge til en pakke i en annen, kan du fra hvor som helst i prosjekte
 Det finnes avhengigheter på ulike nivåer i Jøkul. Globale avhengigheter, for eksempel de som trengs til å kjøre en byggejobb, ligger på rotnivå. Du kan legge til en ny pakke på rotnivå, med kommandoen `yarn add pakke -W`. For å legge til som en utviklingsavhengighet bruker du `-DW`.
 
 Lokale avhengigheter, som trengs for at en pakke skal fungere i bruk, legger vi vanligvis på pakkenivå. Se "Slik lenker du pakker sammen". Bruk kommandoen `yarn add` for å legge til pakker som ikke er en del av Jøkul. Før du gjør det, er det lurt å tenke på om de skal sendes med komponenten eller om brukeren av komponenten har ansvaret for å ha avhengigheten i sitt prosjekt. React bør for eksempel være en `peerDependency`, slik at brukeren ikke ender opp med flere utgaver av React i sin pakke.
+
+
+## Ende til ende tester
+Hvis du lager en ny komponent, så kan du installere Cypress lokalt og utvikle ny integrasjons- og visuell regresjonstest av komponenten din. Når du er fornøyd med testen, sørg for at ingen snapshots blir med i din commit.
```

Comment:
Finnes det en løsning for å gjøre det enklere å ikke få med snapshots? Ser for meg at dette er en tabbe som kan og vil bli begått 🙄

Reply:
Tja, sant. Kan legge inn `__snapshots__` i `.gitignore`, og i workflowen legge til `git add --force cypress`

Reply:
Høres ut som en god idé